docs/bundles/plain.md

```diff
@@ -2,43 +2,147 @@
 
 ## Overview
 
-This document is meant to define the `plain` bundle format as a reference for those publishing `plain` bundles
-for use with RukPak. For more information on the concept of a bundle, click [here](https://github.com/operator-framework/rukpak#bundle).
+This document is meant to define the plain bundle format as a reference for those publishing plain bundles for use with
+RukPak. A bundle is a collection of Kubernetes resources that are packaged together for the purposes of installing onto
+a Kubernetes cluster. A bundle can be unpacked onto a running cluster, where controllers can then create the underlying
+content embedded in the bundle. The bundle can be used as the underlying `spec.source` for
+a [`Bundle`](https://github.com/operator-framework/rukpak#bundle) resource.
+
+A plain bundle is a static collection of arbitrary Kubernetes YAML manifests. A plain bundle is contained in a directory
+that can be packaged in a container image layer , a `git` repository or any other content source that the
+[plain bundle provisioner](../provisioners/plain.md) supports.
+
+Supported source types for a plain bundle currently include the following:
+
+* A directory in a [container image](../sources/image.md)
+* A directory in a [`git` repository](../sources/git.md)
+* A set of keys in a [`ConfigMap`](../sources/local.md)
+* An [upload](../sources/upload.md)
+* A `.tgz` file returned by a [http endpoint](../sources/http.md)
 
-A plain bundle is simply a collection of static, arbitrary, Kubernetes YAML manifests in a given directory.
 
 The currently implemented plain bundle format is the `plain+v0` format. The name of the bundle format, `plain+v0`
-combines the type of bundle (plain) with the current schema version (v0).
+combines the type of bundle (plain) with the current schema version (v0). The
+[plain bundle provisioner](../provisioners/plain.md) is able to source
+`plain+v0` bundles and install them onto a Kubernetes cluster.
 
-> Note: the plain+v0 bundle format is at schema version v0, which means it's an experimental format that is subject
+> Note: the `plain+v0` bundle format is at schema version v0, which means it's an experimental format that is subject
 > to change.
 
-## Example
+## Common Terminology
+
+* *bundle* is a collection of files that define content to be deployed to a cluster
```

Comment:
proposing "manifests" or "manifest files" instead of "files"

Reply:
https://github.com/operator-framework/rukpak/pull/642/commits/902da450e601fa3c1b012ca90b21745fb1ecaa25